### PR TITLE
[Android] Action.tooltip support

### LINKED ISSUE #5469: [Android] Renderer support for tooltip property

### PR DESCRIPTION
**Pull request by golddove**
# Related Issue

Fixes #5469 

# Description

Implement support for Action.tooltip. Additionally, previous implementation was not setting Action.title as accessible name when applied to a `selectAction` or an icon-online `inlineAction`. That is now fixed.

# Sample Card

`v1.5\Tests\TooltipTestCard.json`

# How Verified

Manual verification with Talkback

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5800)

### CODE REVIEW

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java [hunk 1]
```diff
@@ -226,6 +229,17 @@ public static void setSelectAction(RenderedAdaptiveCard renderedCard, BaseAction
...
+            String contentDescription = !TextUtils.isEmpty(selectAction.GetTitle()) ? selectAction.GetTitle() : selectAction.GetTooltip();
+            String tooltip = !TextUtils.isEmpty(selectAction.GetTooltip()) ? selectAction.GetTooltip() : selectAction.GetTitle();
+            if (!TextUtils.isEmpty(tooltip))
+            {
+                view.setContentDescription(contentDescription);
+            }
+            if (!TextUtils.isEmpty(tooltip))
+            {
+                TooltipCompat.setTooltipText(view, tooltip);
+            }
```
Comment (paulcam206) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java:242]: feels like this should probably be pulled out into a helper function

**Review by paulcam206 — CHANGES_REQUESTED** [review]

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java [hunk 2]
```diff
@@ -226,6 +229,17 @@ public static void setSelectAction(RenderedAdaptiveCard renderedCard, BaseAction
             }
 
             view.setOnClickListener(new BaseActionElementRenderer.SelectActionOnClickListener(renderedCard, selectAction, cardActionHandler));
+
+            String contentDescription = !TextUtils.isEmpty(selectAction.GetTitle()) ? selectAction.GetTitle() : selectAction.GetTooltip();
+            String tooltip = !TextUtils.isEmpty(selectAction.GetTooltip()) ? selectAction.GetTooltip() : selectAction.GetTitle();
+            if (!TextUtils.isEmpty(tooltip))
```
[the remark this answers is not in the document]
Comment (golddove) [reply source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java]: Good catch, thanks!

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java (hunk 1 above)
Comment (golddove) [reply source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java]: Refactored

**Review by paulcam206 — APPROVED** [review]
:shipit:

**Pull request merged**